caravel/assets/javascripts/SqlLab/main.css

```diff
@@ -123,7 +123,7 @@ div.Workspace {
     color: black;
 }
 .success {
-    background-color: green;
+    background-color: #4AC15F;
```

Comment:
we should use the bootstrap variable here in case we decide to change the success color: `@brand-success`